warehouse/views.py

```diff
@@ -1,367 +0,0 @@
-from rest_framework.views import APIView
-from rest_framework.response import Response
-from rest_framework import status, generics
-from rest_framework.reverse import reverse
-
-from .models import Product, ShelfBox, Shelf, Transport
-from .api.serializers import ProductSerializer, ShelfSerializer, TransportSerializer
-from .const import *
```

Comment:
Definitywnie polecam bardziej konkretne importy, fakt w tym wypadku jest to mniej znaczące niż przy wielkich rozbudowanych bibliotekach, które posiadają nadmiar narzędzi